Library/Homebrew/cmd/test-bot.rb

```diff
@@ -639,10 +639,9 @@ def homebrew
         tests_args << "--coverage" if ruby_two && ENV["TRAVIS"]
         test "brew", "tests", *tests_args
         test "brew", "tests", "--no-compat"
+        test "brew", "readall", "--syntax" if ruby_two
       elsif @tap.core_tap?
-        readall_args = []
-        readall_args << "--syntax" if ruby_two
-        test "brew", "readall", "--aliases", *readall_args
+        test "brew", "readall", "--aliases"
       else
         test "brew", "readall", @tap.name
```

Comment:
Could make sense to include `--aliases` here as well, as the alias verification also correctly works with taps (though most taps probably don't even have an `Aliases/` directory—none of the official non-core taps currently have).